templates/html-header.twig

```diff
@@ -5,6 +5,28 @@
 	<title>{% if wp_title %}{{ wp_title|e('esc_html')|raw }} - {{ site.name }}{% else %}{{ site.name }}{% endif %}</title>
 	<meta name="description" content="{{ site.description }}">
 
+	{% if post %}
+		{% set description   = post.post_excerpt %}
+		{% set attachment_id = post.background_image_id ?? fn('get_post_thumbnail_id', post) %}
```

Comment:
@angtheod Just a question - can we move `post.background_image_id ?? fn('get_post_thumbnail_id', post) %}`  this logic to `page.php`?

Reply:
@sagarsdeshmukh Why should we do this?

Reply:
I think we can keep the conditional part in page.php